### PR TITLE
fix: allow -q by turning of pipe progress bar

### LINKED ISSUE #4823: `mc pipe -q` displays progress bar
## Expected behavior

`mc pipe -q` should not display progress bar

## Actual behavior

`mc pipe -q` displays progress bar

image

## Steps to reproduce the behaviour

Run `echo bla | ./mc pipe -q bt/sd/deleteme-andrew` 

## mc --version
```
mc version RELEASE.2024-01-13T08-44-48Z (commit-id=799ded568dc7a40547e417be460de37b985ab11a)
Runtime: go1.21.5 linux/amd64
Copyright (c) 2015-2024 MinIO, Inc.
License GNU AGPLv3 <https://www.gnu.org/licenses/agpl-3.0.html>
```

## System information
Debian

### PR DESCRIPTION
**Pull request by jiuker**
## Community Contribution License
All community contributions in this pull request are licensed to the project maintainers
under the terms of the [Apache 2 license](https://www.apache.org/licenses/LICENSE-2.0). 
By creating this pull request I represent that I have the right to license the 
contributions to the project maintainers under the Apache 2 license.

## Description

fix: pipe quiet -q dot not work
before: -q will not work.
after: -q will work.
```log
MacBook-Pro mc % echo 123 | ./mc pipe minio/mytest/123q.txt 
 0 B / ? ━┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉━━
MacBook-Pro mc % echo 123 | ./mc pipe minio/mytest/123q.txt -q
MacBook-Pro mc % 

```

## Types of changes
- [x] Bug fix #4823
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Optimization (provides speedup with no functional changes)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
- [ ] Fixes a regression (If yes, please add `commit-id` or `PR #` here)
- [ ] Unit tests added/updated
- [ ] Internal documentation updated
- [ ] Create a documentation update request [here](https://github.com/minio/docs/issues/new?label=doc-change,title=Doc+Updated+Needed+For+PR+github.com%2fminio%2fmc%2fpull%2fNNNNN)

### CODE REVIEW

File: cmd/pipe-main.go [hunk 1]
```diff
@@ -183,6 +189,9 @@ func mainPipe(ctx *cli.Context) error {
 	// validate pipe input arguments.
 	checkPipeSyntax(ctx)
 
+	// globalQuiet is true for no window size to get. We just need --quiet here.
+	quiet := ctx.IsSet("quiet")
```
Comment (shtripat) [inline cmd/pipe-main.go:193]: We should add `--quiet` to command line options as well at Line#37. Also add help content about this flag.

**Review by harshavardhana — APPROVED** [review]

File: cmd/pipe-main.go (hunk 1 above)
Comment (jiuker) [reply cmd/pipe-main.go:193]: this flag is global flag @shtripat

Comment (shtripat) [reply cmd/pipe-main.go:193]: ah thats my bad. thanks

**Review by shtripat — APPROVED** [review]
LGTM. Verified

**Review by klauspost — APPROVED** [review]

**Pull request merged**